### DISCUSSION TITLE
cannot build phi2 model on system with 32 GB ram

### DISCUSSION THREAD

**Discussion by liqunfu** [microsoft/onnxruntime-genai#489] [Performance improvements]
python -m onnxruntime_genai.models.builder -m microsoft/phi-2 -e cpu -p int4 --extra_options int4_accuracy_level=4  -o ./example-models/phi2-int4-cpu

the system memory usage keep increasing up to 30 GB and it exit without any message.

**Comment by natke** [comment]
Hi @liqunfu, can you share which version of onnxruntime-genai you are using and your O/S and machine specs please.

**Comment by liqunfu** [comment]
> Hi @liqunfu, can you share which version of onnxruntime-genai you are using and your O/S and machine specs please.

@kunal-vaishnavi , I was working with Kunal when finding this bug.

**Comment by natke** [comment]
Could not repro this issue. Please get in touch if it persists
